pkg/proxy/proxy.go

```diff
@@ -624,6 +655,64 @@ func (s *Proxy) generateToken() string {
 	return fmt.Sprintf("%x", b)
 }
 
+// WatchSVC when external svc becomes active update cluster info
+func (s *Proxy) WatchSVC() (watch.Interface, error) {
+	klog.Infof("WatchSVC start")
+	watcher, err := s.k8sClientSet.CoreV1().Services(defaultAgentNamespace).Watch(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		klog.Errorf("start WatchSVC error:%v", err)
+		return nil, err
+	}
+	go func() {
+		for event := range watcher.ResultChan() {
+			svc := event.Object.(*v1.Service)
+			switch event.Type {
+			case watch.Added:
+			case watch.Modified:
+				klog.Infof("service:%s found", svc.Name)
+				if !strings.HasPrefix(svc.Name, svcPrefix) {
+					klog.Infof("service:%s has no relation with tower", svc.Name)
```

Comment:
Always use klog.Info with a level, otherwise, prepare for the flooding :).  https://github.com/kubesphere/community/blob/master/developer-guide/development/coding-conventions.md